### DISCUSSION TITLE
Welcome to Layui Discussions!

### DISCUSSION THREAD

**Discussion by sentsim** [layui/layui#1456] [动态]
## 👋 欢迎 Layui 全体开发者!

  我们正式采用 Github 的 Discussions 作为论坛区，供大家进行技术交流。我们希望您能：
  * 提出您想知道的问题（ 如果是 Bug 和 Feature Request，则要发到 Issues 区）。
  * 分享您的想法。
  * 与其他社区成员进行互动。
  * 保持开放的心态，本着相互尊重、理解和友善的态度，共同维护 Layui 来之不易的良好的社区氛围 💖。<br>请记住，这是一个我们共建的社区 💪。

欢迎随时在评论区留言。

> 附录：
> 1. 开通 Discussions 的缘由，详见：[📣 为什么要将 Layui Issues 由 Gitee 转移到 Github](https://gitee.com/layui/layui/issues/I8OVIB)
> 2. 关于 Discussions 与 Issues 的区别，参考：🍀 [Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

---

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ByYogi** [comment]
能不能用写个中文版的

**Reply by sentsim** [reply]
已写。
